packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -9,101 +9,148 @@ ouia: true
 import pfLogo from './pfLogo.svg';
 import pfLogoSmall from './pf-logo-small.svg';
 
-## Examples
 
-### Basic
+## Examples 
+
+### Basic cards
+
+Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`, though only a body is required. 
```

Comment:
Technically `<CardBody>` isn't required. Those sections add padding and spacing and stuff. If you just want an outer card wrapper with custom stuff in it (no padding), just use `<Card>` on its own. Here's an example - the data list in the page section is in an empty `<Card>` - https://www.patternfly.org/v4/components/data-list/html-demos/basic/

Reply:
Is it worth mentioning why we generally recommend including CardBody, or do you think it's enough to just say we recommend at least including that? If so - is the reasoning just because it applies better practice styling to content? 

In the demo is the empty <Card> the one that just says "patternfly-unified-design-kit"? And this may be a weird question, but is there any way to see the js code for the cards in the demo, like the code snippets that are available in the examples? For example, can you see the specific card implementation within Chrome's devtools for the demo? Kind of just curious 😆

Reply:
@edonehoo For the code for that demo, it's only in the HTML Demos tab but: https://www.patternfly.org/v4/components/data-list/html-demos#basic

Essentially everything below the "This is a full page demo." text on that demo is inside a Card.

In terms of what should be mentioned... Since this verbiage is on a "basic card" example (and probably not intended for a case like this demo), I could see something along the lines of "For typical use-cases, it's recommneded to at least include a CardBody to provide details about the card item" possibly working if we want to include that sort of verbiage. An alternative would be to just omit that part and simply state "Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`."